core/foundation/res/ROOT/FoundationUtils.hxx

```diff
@@ -39,6 +39,30 @@ namespace FoundationUtils {
    std::string MakePathRelative(const std::string &path, const std::string &base,
                                 bool isBuildingROOT = false);
 
+   ///\returns the path separator slash or backslash depending on the platform.
+   inline const std::string& GetPathSeparator() {
+#ifdef WIN32
+      static const std::string gPathSeparator ("\\");
```

Comment:
at least in the past, we had challenge on windows with inline function and static objects.

Reply:
Would the bots catch it?